commit b925be726bdd44ae3cd909fd0cd90633bf5c7de5
Author: Sage Weil <sweil@redhat.com>
Date:   Sun Mar 10 10:48:21 2019 -0500

    mgr/MgrStandby: exit with 0 result code, as if we'd done an orderly shutdown
    
    This matches the prior behavior with shutdown().  It also makes
    teuthology happy.
    
    Signed-off-by: Sage Weil <sage@redhat.com>

diff --git a/src/mgr/MgrStandby.cc b/src/mgr/MgrStandby.cc
index bf6dc65f68d..ee246b8cbf6 100644
--- a/src/mgr/MgrStandby.cc
+++ b/src/mgr/MgrStandby.cc
@@ -264,7 +264,7 @@ void MgrStandby::handle_signal(int signum)
 {
   ceph_assert(signum == SIGINT || signum == SIGTERM);
   derr << "*** Got signal " << sig_str(signum) << " ***" << dendl;
-  _exit(128 + signum);
+  _exit(0);  // exit with 0 result code, as if we had done an orderly shutdown
   //shutdown();
 }
 
